deploy/canary/Dockerfile

```diff
@@ -3,8 +3,7 @@ MAINTAINER vmarmol@google.com
 
 RUN apt-get install -y git dmsetup
```

Comment:
This is for our jenkins automation that builds a canary docker image from HEAD every night. We can update it I guess to use existing make rules. Again can we do that in a separate PR?